_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -17339,15 +17339,8 @@
 /turf/open/floor/iron/dark,
 /area/command/heads_quarters/cmo)
 "dKB" = (
-/obj/structure/window/reinforced{
-	dir = 4
-	},
-/obj/effect/turf_decal/stripes/line{
-	dir = 4
-	},
-/obj/effect/decal/cleanable/dirt,
-/turf/open/floor/iron,
-/area/maintenance/disposal)
+/turf/closed/wall,
+/area)
```

Comment:
you replaced /area/maintenance/disposal with just "/area" ... why and how?

Reply:
Spam deleting the top layer because I placed too many things on accident, fixed in the next version coming